flexget/plugins/input/npo_watchlist.py

```diff
@@ -267,6 +270,8 @@ def _parse_episodes_std(self, task, config, series_name, page, series_info):
             e['npo_name'] = series_info['npo_name']
             e['npo_description'] = series_info['npo_description']
             e['npo_language'] = series_info['npo_language']
+            if subtitle[1]:  # if runtime is available in the subtext
```

Comment:
The previous split might indeed fail, because there is no easy way to check if 100% of the site confirms to this new subtext format. I will add an extra check